docs/content.go

```diff
@@ -176,9 +175,8 @@ Enterprise Performance Computing (EPC)`
 	CacheCleanUse   string = `clean [clean options...]`
 	CacheCleanShort string = `Clean your local Singularity cache`
 	CacheCleanLong  string = `
-  This will clean you local cache: "${HOME}/.singularity/cache". The available cache
-  types are: library, oci, and blob. By default cache clean will only clean blob cache,
-  use: '--all' to clean all cache.`
+  This will clean your local cache (stored at $HOME/.singularity/cache if SINGULARITY_CACHEDIR is not set).
+  By default only blob cache is cleaned, use '--all' to clean all cache.`
```

Comment:
Maybe change it a bit more...

```suggestion
  By default only oci blobs cache is cleaned, use '--all' to clean all cache.`
```

Reply:
We already removed `oci blob` phrase [here](https://github.com/sylabs/singularity/pull/3073) so let's not add it here as well for consistency.